read_mars/__init__.py

```diff
@@ -42,3 +42,35 @@ def read_mars(filename, tree='Events', verbose=False):
     f.Close()
 
     return pd.DataFrame(events)
+
+
+def read_mars_fast(filename, tree='Events', leaves=[]):
+    """Return a Pandas DataFrame of a star or ganymed output root file.
+    
+    A faster (~factor 15) version of read_mars. It also omits the useless leaves fBits and fUniqueID. 
+    Keyword arguments:
+    tree -- Set, which tree to read. (Default = "Events")
+    leaves -- Specify a list of leaves. (Default is [], what reads in all leaves)
+    """
+
+    f = ROOT.TFile(filename)
+    tree = f.Get(tree)
+    if not leaves:
+        leaves = [l.GetName() for l in tree.GetListOfLeaves() if
+            not (l.GetName().endswith('.') or l.GetName().endswith('fBits') or l.GetName().endswith('fUniqueID'))]
+
+    n_events = tree.GetEntries()
+    tree.SetEstimate(n_events + 1) #necessary for files with more than 1 M events
+    df = pd.DataFrame(np.empty([n_events, len(leaves)]), columns=leaves)
+    b = np.empty([n_events])
+
+    for leaf in leaves:
+        tree.Draw(leaf, "", "goff")
+        v1 = tree.GetV1()
+        v1.SetSize(n_events + 1)
```

Comment:
This seems to me like a lot of ROOT magic. Why do we Draw the Tree? What does the `goff` drawing option do? 
What is V1?
This kind of should be explained in comments, I guess.